Bhagyashri/Correction/css/style.css

```diff
@@ -0,0 +1,109 @@
+
+body{
+background-color:#006966;
+padding-left:100px;
+padding-right:100px;
+margin:0px;
+}
+
+.first-sect1{
+	   border-top:0px;
+	   padding:10px;
+       background-color:#ffffff;
+	   min-height:40px;
+           }
+		   
+.class-hg1{
+	float:left;
+	display:inline-block;
+	  }
+	  .class-hg1 img{  
+	  width:150px;
+	  }
+.class-hg2{
+	text-transform:uppercase;
+	float:right;
+	display:inline-block;
+	}
+nav{
+	color:#008080;
+	padding:10px;
+}	
+
+a{
+	text-decoration:none;
+	padding-left:10px;
+	padding-right:10px;
+ }
+ .class-h3{
+	padding-top:30px;
+	padding-bottom:30px;
+ }
+
+.class-h31{
+
+	margin-bottom:4px;
+	text-align:center;
+	font-size=20px;
+	color:#ffffff;
+	
+}
+.class-h32{
+
+	padding-top:0px;
+	border-top:0px;
+	margin-top:0px;
+    text-align:center;
+	font-size=20px;
+	color:#ffffff;
+}
+
+h1{
+	color:#008080;
+	font-size:30px;
+	padding:0px;
+	border:0px;
+	margin-top:0px;
+	margin-bottom:9px;
+}
+h2{
+	color:#ffffff;
+	margin-top:1px;
+	padding:0px;
+	border:0px;
+  }
+.class-left{
+float:left;
+width:40%
+	}
+p{
+   color:#009999;
+ }
+
+ul{
+	color:#009999;
+	margin-bottom:50px;
+	padding-inline-start:0px;
+  }
+button{
+	padding:10px 35px;
+	border-width:0px;
+	border-style:normal;
+	border-color:none;
+	background-color:#007070;
+    background-color:rgba(00,70,70,0.2);
+    color:rgba(00,40,40);
+     }
+                
+.class-first{
+	 min-height:90px;
+	 Position:relative;
+}
+	 
+.div-right img{
+     height:300px;
+	 width:500px;
```

Comment:
for image you can add width: 100%; and add min-height: 300px;
don't add fix height.

Reply:
Have given
 min-height:70px;
	 width:60%;

if write width:100%
then image is not fixed at mentioned position.